templates/taxi/car_detail.html

```diff
@@ -3,20 +3,33 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
   </h1>
+  {% if car in user.cars.all %}
```

Comment:
maybe refactor this to if `user in car.drivers.all` and add prefetch on the view to fix n+1 problem